apple/Sources/FerrostarSwiftUI/Views/SpeedLimit/USSpeedLimitView.swift

```diff
@@ -54,20 +54,21 @@ public struct USStyleSpeedLimitView: View {
             .background(Color.white)
             .frame(width: 52, height: 76)
             .cornerRadius(4)
+            .colorScheme(.light)
```

Comment:
This fixes the speed limit sign colors to light mode. Since it's something we don't want to invert.

Reply:
Leaving a comment for posterity because this was EXTREMELY interesting... Apparently named colors (ex: `.green`) may be aware of light/dark mode, and resolve to different hex values! That's both unexpected but also kinda cool.